### ISSUE TITLE
Notes are not captured during translation

### ISSUE THREAD

**Issue by wip-abramson** [LegReq/xmi-mermaid#2]
I have a bunch of ntoes on the sequence diagrams in VP. These are not currently being reproduced in mermaid

**Comment by jandrieu** [comment]
Try this
https://github.com/mermaidjs/mermaid-gitbook/blob/master/content/sequenceDiagram.md#notes

**Comment by wip-abramson** [comment]
Yeah, adding notes is fine and I guess I can do it manually. But ideally we would update this script, so that it notices notes from the VP xmi and translates them into mermaid

**Comment by jandrieu** [comment]
Yep.  It's a good feature to prioritize
